src/OpenTelemetry.Instrumentation.SqlClient/SqlClientInstrumentationOptions.cs

```diff
@@ -154,6 +156,34 @@ internal void AddConnectionLevelDetailsToActivity(string dataSource, Activity sq
             }
         }
 
+        internal void AddDBUserToActivity(string connectionString, Activity sqlActivity)
+        {
+            if (!this.EnableConnectionLevelAttributes)
+            {
+                sqlActivity.SetTag(SemanticConventions.AttributeDbConnectionString, connectionString);
```

Comment:
Sql connection string should probably be explicit op-in like command text because it can contain passwords.

Reply:
Totally make senses, we can remove the `password` section when there is one

Reply:
Thanks @lupengamzn!